app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -608,6 +608,25 @@ def test_ending_experiment_completed_when_record_is_not_in_main(self):
             ).exists()
         )
 
+    def test_updating_experiment_with_published_dto_none_is_skipped(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.PAUSING_APPROVE_WAITING,
+            application=NimbusExperiment.Application.DESKTOP,
+            published_dto=None,
+        )
```

Comment:
A new test?

Reply:
One of the things the typechecker gets you is looking for where things could be `None` but the code isn't handling the `None` case and then throws a big stink about it, which is good.  So now there's a bunch of cases where we check for `None` that were never covered by existing tests, to then the coverage checker will say 'these lines aren't tested', so you gotta add a new test.  This is all part of the dream of having static type analysis in Python 🙏